client/src/components/Common/BookingForm/BookingForm.js

```diff
@@ -4,34 +4,45 @@ import Clock from "../../../Images/clock_icon.png";
 import RoomIcon from "../../../Images/room_icon.png";
 import "./bookingForm.css";
 import axios from "axios";
-import { Link } from "react-router-dom";
 import Popup from "./Popup";
+import moment from 'moment'
+
 class BookingForm extends Component {
   constructor(props) {
-    super(props);
+    super(props); 
     this.state = {
       userId: 3,
-      date: "2020-05-23T11:00:00.000Z",
+      date: '',
       roomId: 5,
```

Comment:
should read the room Id from props with the room name

Reply:
I forget that. Hmm so we should put the navbar in userhome component so user id can be passed to rooms component then to bookingForm component @yaqootturman